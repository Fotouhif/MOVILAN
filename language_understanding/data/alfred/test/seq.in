Turn left and face the desk to the left of the blue rubbishbin 
Pick up the yellow pencil to the rear of the CD on the top shelf of the desk 
Back away from the desk slightly 
Place the pencil in the glass bowl to the right of the mug on the desk 
Pick up the glass bowl containing a pencil from the desk 
Turn around and towards the laundrybasket then turn left and move to the nightstand in front of the window 
Place the glass bowl containing a pencil to the left of the CD on the nightstand 
Turn around and go to the front of the counter on the left of the stove 
Pick up the gray knife behind the saltshaker 
Turn around and go to the left side of the white table on the left 
Slice the lettuce on the table 
Turn right and go to the refrigerator on the left 
Place the knife in the refrigerator 
Turn around and go back to the left side of the white table with the slicedlettuce on top 
Pick up one end slice of the lettuce 
Turn right and go back to the refrigerator on the left 
Chill the lettuce in the refrigerator and take it out 
Turn around and go back to the white table on the right 
Place the lettuce on the table in the middle 
Turn left and walk to the white table with the plant on it on the left 
Pick up the pen to the right of the plant 
Turn around and walk to the television on the left 
Put the pen down on the televisionstand to the left of the remote 
Walk to the couch turn left walk straight ahead to the wooden tables 
Pick up the pen from the left wooden table 
Turn around and walk to the televisionstand 
Put the pen down on the televisionstand and on top of the remote on the right 
Walk straight towards the desk 
Pick up the pen to the right of the alarmclock 
Take a step backwards from where you are at 
Open the bottom drawer and place the pen in it 
walk to face table to the left of the fridge 
pick up spatula from the table 
put spatula in white bowl on table 
pick up bowl from table 
walk left to face counter to the left of the fridge 
put the bowl on the counter 
Turn to your left and approach the sink 
Pick up the spraybottle from the sink 
Turn around and face the dresser to your left 
Place the spraybottle on the dresser 
Turn around and walk towards the sink 
Pick up the spraybottle from the cabinet under the sink 
Turn around and walk back towards the dresser 
Place the spraybottle on top of the dresser 
Turn around and step forward to face the counter 
Pick up the knife on the counter 
Turn around and walk across the room to face the bread on the counter 
Cut the bread on the counter into slices 
Turn left and step forward to face the microwave above the stove