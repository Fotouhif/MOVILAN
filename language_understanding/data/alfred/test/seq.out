action_navi action_desc O target_rel O target_obj O O refinement_rel O O refinement_attri refinement_obj
action_n_navi action_desc O target_attri target_obj O O refinement_rel O O refinement_obj refinement_rel O refinement_rel refinement_obj refinement_rel O refinement_obj
action_navi action_desc O O target_obj action_intensity
action_n_navi O target_obj refinement_rel O refinement_attri refinement_obj O O refinement_rel O O refinement_obj refinement_rel O refinement_obj
action_n_navi action_desc O target_attri target_obj O O refinement_obj refinement_rel O refinement_obj
action_navi action_desc O action_desc O target_obj O action_navi action_desc O action_navi O O target_obj O refinement_rel O O refinement_obj
action_n_navi O target_attri target_obj O O refinement_obj O O refinement_rel O O refinement_obj refinement_rel O refinement_obj
action_navi action_desc O action_navi O O refinement_rel O O refinement_obj O O refinement_rel O O refinement_obj
action_n_navi action_desc O target_attri target_obj refinement_rel O refinement_obj
action_navi action_desc O action_navi O O target_rel O O O target_attri target_obj O O agent_dir
action_n_navi O target_obj refinement_rel O refinement_obj
action_navi action_desc O action_navi O O target_obj O O agent_dir
action_n_navi O target_obj refinement_rel O refinement_obj
action_navi action_desc O action_navi action_desc O O target_rel O O O target_attri target_obj refinement_rel O refinement_obj O O
action_n_navi action_desc O O target_obj O O target_obj
action_navi action_desc O action_navi action_desc O O target_obj O O agent_dir
action_n_navi O target_obj refinement_rel O refinement_obj O action_n_navi O action_desc
action_navi action_desc O action_navi action_desc O O target_attri target_obj O O agent_dir
action_n_navi O target_obj refinement_rel O refinement_obj O O refinement_rel
action_navi action_desc O action_navi O O target_attri target_obj refinement_rel O refinement_obj target_rel O O O agent_dir
action_n_navi action_desc O target_obj O O refinement_rel O O refinement_obj
action_navi action_desc O action_navi O O target_obj O O agent_dir
action_n_navi O target_obj action_desc refinement_rel O refinement_obj O O refinement_rel O O refinement_obj
action_navi O O target_obj action_navi action_desc action_navi action_desc action_desc O O target_attri target_obj
action_n_navi action_desc O target_obj refinement_rel O refinement_rel refinement_attri refinement_obj
action_navi action_desc O action_navi O O target_obj
action_n_navi O target_obj action_desc refinement_rel O refinement_obj O refinement_rel refinement_rel O O refinement_obj O O agent_dir
action_navi action_desc action_desc O target_obj
action_n_navi action_desc O target_obj O O refinement_rel O O refinement_obj
action_n_navi O action_intensity action_desc O O O O O
action_n_navi O refinement_rel refinement_obj O action_n_navi O target_obj refinement_rel O
action_navi O target_rel target_obj O O refinement_rel O O refinement_obj
action_n_navi action_desc target_obj refinement_rel O refinement_obj
action_n_navi target_obj O refinement_attri refinement_obj refinement_rel refinement_obj
action_n_navi action_desc target_obj refinement_rel refinement_obj
action_navi action_desc O target_rel target_obj O O refinement_rel O O refinement_obj
action_n_navi O target_obj refinement_rel O refinement_obj
action_navi O O action_desc O O O target_obj
action_n_navi action_desc O target_obj refinement_rel O refinement_obj
action_navi action_desc O target_rel O target_obj O O agent_dir
action_n_navi O target_obj refinement_rel O refinement_obj
action_navi action_desc O action_navi action_desc O target_obj
action_n_navi action_desc O target_obj refinement_rel O refinement_obj refinement_rel O refinement_obj
action_navi action_desc O action_navi action_desc action_desc O target_obj
action_n_navi O target_obj refinement_rel refinement_rel O O refinement_obj
action_navi action_desc O action_navi action_desc O target_rel O target_obj
action_n_navi action_desc O target_obj refinement_rel O refinement_obj
action_navi action_desc O action_navi action_desc O target_obj O target_rel O target_obj refinement_rel O refinement_obj
action_n_navi O target_obj refinement_rel O refinement_obj O O
action_navi action_desc O action_navi action_desc O target_rel O target_obj refinement_rel O refinement_obj